solidity/contracts/igps/InterchainGasPaymaster.sol

```diff
@@ -11,20 +12,25 @@ import {OwnableUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Own
  * @notice Manages payments on a source chain to cover gas costs of relaying
  * messages to destination chains.
  */
-contract InterchainGasPaymaster is IInterchainGasPaymaster, OwnableUpgradeable {
+contract InterchainGasPaymaster is
+    IInterchainGasPaymaster,
+    IGasOracle,
+    OwnableUpgradeable
+{
+    /// @notice The scale of gas oracle token exchange rates.
+    uint256 internal constant TOKEN_EXCHANGE_RATE_SCALE = 1e10;
```

Comment:
should this be applied outside `GasOracle`s?

Reply:
sorry wdym here?